scrapscript.py

```diff
@@ -206,11 +211,21 @@ def read_comment(self) -> None:
             pass
 
     def read_number(self, first_digit: str) -> Token:
+        # TODO: Support floating point numbers with no integer part
```

Comment:
There's also e notation/scientific notation to think about, but I think that can be a separate PR